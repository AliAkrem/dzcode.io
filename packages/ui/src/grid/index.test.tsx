import { render } from "@testing-library/react";

import { Grid } from ".";

it("should render Grid component", () => {
  const { container } = render(<Grid />);
  expect(container).toMatchSnapshot();
});
