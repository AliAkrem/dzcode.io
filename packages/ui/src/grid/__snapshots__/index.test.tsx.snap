// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Grid component 1`] = `
<div>
  <div
    class="MuiGrid-root css-vj1n65-MuiGrid-root"
  />
</div>
`;
